mappings/net/minecraft/client/gui/screen/world/CreateWorldScreen.mapping

```diff
@@ -4,6 +4,12 @@ CLASS net/minecraft/class_525 net/minecraft/client/gui/screen/world/CreateWorldS
 	FIELD field_24288 gameRules Lnet/minecraft/class_1928;
 	FIELD field_24588 moreOptionsDialog Lnet/minecraft/class_5292;
 	FIELD field_25478 dataPacksButton Lnet/minecraft/class_4185;
+	FIELD field_25898 GAME_MODE_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26598 ENTER_SEED_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26599 SEED_INFO_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26600 ENTER_NAME_TEXT Lnet/minecraft/class_2561;
+	FIELD field_26601 RESULT_FLOOR_TEXT Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	FIELD field_26601 RESULT_FOLDER_TEXT Lnet/minecraft/class_2561;
```

Reply:
Fixed in b79ddfbfb61242b790ce49b0a387c3d27a8e56f9